eth_utils/functional.py

```diff
@@ -2,23 +2,33 @@
 import functools
 import itertools
 
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+)
+
 from .toolz import (
     compose as _compose,
 )
 
 
-def identity(value):
+AnyCallable = Callable[..., Any]
```

Comment:
Thoughts on this? Not really necessary/doing anything useful, but might help the readability